code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -97,6 +97,10 @@
 	desc = "A wicked machine used by those skilled in magical arts. It is inactive."
 
 /obj/structure/constructshell/attackby(obj/item/O, mob/user, params)
+	if(!iscultist(user) && !iswizard(user))
+		user << "<span class='danger'>An overwhelming feeling of dread comes over you as you pick up the soulstone. It would be wise to be rid of this quickly.</span>"
+		user.Dizzy(120)
```

Comment:
Wouldn't that let the non-cultist/wizard make the construct THEN give them the message about being dizzy? 

The change was to prevent non-cultists from grabbing a filled shard and making a juggernaut by accident (this actually happened 2 days ago).